x-pack/plugins/cases/public/common/use_cases_toast.tsx

```diff
@@ -47,10 +62,11 @@ function getToastTitle({
     return title;
   }
   if (attachments !== undefined) {
-    for (const attachment of attachments) {
-      if (attachment.type === CommentType.alert) {
-        return CASE_ALERT_SUCCESS_TOAST(theCase.title);
-      }
+    const alertsCount = getAlertsCount(attachments);
+    if (alertsCount === 1) {
+      return CASE_ALERT_SUCCESS_TOAST(theCase.title);
+    } else if (alertsCount > 1) {
+      return CASE_ALERT_PLURAL_SUCCESS_TOAST(theCase.title);
```

Comment:
It is possible to pluralize translations, so we don't have to add a new one for each plural:
```
export const CASE_ALERT_SUCCESS_TOAST = (title: string, quantity: number = 1) =>
  i18n.translate('xpack.cases.actions.caseAlertSuccessToast', {
    values: { quantity, title },
    defaultMessage: '{quantity, plural, =1 {An alert has} other {Alerts have}} been added to "{title}"',
  });
```

Reply:
I knew this existed! thanks. I'll change it.